packages/contracts/test/core/dao.ts

```diff
@@ -553,53 +553,29 @@ describe('DAO', function () {
   });
 
   describe('ERC1967', async () => {
-    it('reverts if `UPGRADE_DAO_PERMISSION` is not granted or revoked', async () => {
-      const iface = new ethers.utils.Interface(DAO__factory.abi);
-      const initData = iface.encodeFunctionData('initialize', [
-        dummyMetadata1,
-        ownerAddress,
-        dummyAddress1,
-      ]);
-
-      const ERC1967 = await ethers.getContractFactory('ERC1967Proxy');
-      const erc1967Proxy = await ERC1967.deploy(dao.address, initData);
-
-      const daoProxyContract = dao.attach(erc1967Proxy.address);
-
-      await expect(daoProxyContract.upgradeTo(dao.address))
-        .to.be.revertedWithCustomError(dao, 'Unauthorized')
-        .withArgs(
-          daoProxyContract.address,
-          daoProxyContract.address,
-          ownerAddress,
+    // TODO: Must be made as a test utils that can be imported in every upgradeable test file
+    // Such as https://github.com/OpenZeppelin/openzeppelin-contracts/blob/a28aafdc85a592776544f7978c6b1a462d28ede2/test/token/ERC20/ERC20.behavior.js#L5
+    // This will avoid having the same 3 tests in every file or we could just neglect this test as
+    // It's coming from UUPSUpgradeable which is already tested though since contracts are very critical,
+    // Still testing this most important part wouldn't be bad..
+    it.skip('reverts if `UPGRADE_DAO_PERMISSION` is not granted or revoked', async () => {
```

Comment:
Why skip? Is this part of the audit changes?

Reply:
NO. The reason we skip it is it would be much better if the very same test ends up in upgradeable contracts' test files. For this, we need a common test in separate file which we import and put it in tests. 

Didn't want to fully dive into it due to other issues being a priority.

Reply:
No problem, I created APP-1553 to track this :+1: